src/components/form/Lookup/Film/FilmField.tsx

```diff
@@ -0,0 +1,22 @@
+import { FilmLookupItem } from '.';
+import { LookupField } from '../..';
+import { CreateFilmInput } from '../../../../api';
+import { CreateFilm } from '../../../../scenes/Engagement/LanguageEngagement/Product/Producibles/Film/CreateFilm';
+import { useFilmLookupLazyQuery } from './FilmLookup.generated';
+
+export const FilmField = LookupField.createFor<FilmLookupItem, CreateFilmInput>(
+  {
+    resource: 'Film',
+    useLookup: useFilmLookupLazyQuery,
+    // option is nullish initially
+    // eslint-disable-next-line @typescript-eslint/no-unnecessary-condition
+    getOptionLabel: (option) => option?.name?.value ?? '',
```

Comment:
Why is it called with a null? Sounds like a problem with our recent changes.

Reply:
Ah. Because we call `getOptionLabel` with `null` if multiple is false

```typescript
const selectedText = multiple
    ? ''
    : (getOptionLabel || identity)(field.value as T) ?? '';
```

Reply:
What is the purpose of identity here?

Reply:
Why not:

```typescript
const selectedText =
    !multiple && getOptionLabel && (field.value as T)
      ? getOptionLabel(field.value as T)
      : '';
```

Reply:
Don't you mean this?

```typescript
const selectedText = multiple || !field.value
    ? ''
    : (getOptionLabel || identity)(field.value as T) ?? '';
```

Otherwise the problem with `option` being null in `getOptionLabel` remains. 
`getOptionLabel: (option) => option.name.value ?? ''`

And if the above is correct, `field.value` will never being nullish, but either a string literal or empty string, so the coalescing at the end is unnecessary.

Reply:
Although, we still need to coalesce if `name.value` doesn't exist since it's a secured string, right?

Reply:
You're right, my bad.

Reply:
No problemo